### ISSUE TITLE
Alerts are not recognized when they include a title

### ISSUE THREAD

**Issue by mkoppmann** [lunet-io/markdig#809]
Obsidian allows [adding titles to alerts/callouts](https://help.obsidian.md/Editing+and+formatting/Callouts#Change+the+title). When Markdig parses an alert with a title, it is recognized as a `QuoteBlock` instead of an `AlertBlock`. It would be nice if Markdig could support the optional title.

```markdown
> [!NOTE] Custom Title
> Lorem Ipsum
```

The behavior is consistent with the way GitHub displays alerts with titles though:

> [!NOTE] Custom Title
> Lorem Ipsum

**Comment by xoofx** [comment]
> The behavior is consistent with the way GitHub displays alerts with titles though:

Yes, markdig implements the official GitHub Alert. You can GitHub to add support to title maybe [here](https://github.com/orgs/community/discussions/16925)

**Comment by szalkerous** [comment]
I noticed the Typora editor creates these as well and they aren't handled in many parsers. I'd love to see this implemented, either natively or via a plugin.
